docs/notes/developers/faq.rst

```diff
@@ -88,6 +88,43 @@ concerns, please feel free to `open an issue`_ or `post it to discussions`_.
   Although data migration incurs a cost, the scheduler plugin has been developed to
   prioritize *h2* when launching *B2*, minimizing the need for migration whenever possible.
 
+9. *What's the minimal Kubernetes version requirement for vineyard operator?*
+
+  At present, we only test the vineyard operator based on Kubernetes 1.24.0. 
+  So we highly recommend using Kubernetes 1.24.0 or above.
+
+10. *Why the vineyard operator can't be deployed on Kubernetes?*
+
+  If you deploy the vineyard operator directly on Kubernetes, you may encounter the 
+  following error:
+
+  .. code:: shell
+
+      resource mapping not found for name: "vineyard-serving-cert" namespace: "vineyard-system" from "STDIN": no matches for kind "Certificate" in version "cert-manager.io/v1"
+      ensure CRDs are installed first
+      resource mapping not found for name: "vineyard-selfsigned-issuer" namespace: "vineyard-system" from "STDIN": no matches for kind "Issuer" in version "cert-manager.io/v1"
+      ensure CRDs are installed first
+
+  It is because the vineyard operator depends on the `cert-manager`_ to 
+  generate the TLS certificate for the vineyard cluster. So you need to install the 
+  cert-manager first.
+
+11. *How to connect to the vineyard cluster deployed by the vineyard operator?*
+
+  There are two ways to connect to the vineyard cluster deployed by the vineyard operator:
+  - `Through IPC`. Create a pod with the specific labels so that the pod can be scheduled 
+    to the node where the vineyard cluster is deployed.
+  - `Through RPC`. Connect to the vineyard cluster through the RPC service exposed by the 
+    vineyard operator. You could refer to the `guide`_ for more details.
+
+  Is there a way to install the vineyard cluster on Kubernetes quickly?
+  ----------------------------------------------------------------------
```

Comment:
Why a title line here? Can you use **xxx** for bold instead?

Reply:
It's my bad. I forgot to update it. Thanks for catching this and addressed.